src/pages/resultsView/mutualExclusivity/MutualExclusivityUtil.tsx

```diff
@@ -41,6 +41,22 @@ export function calculateAdjustedPValue(pValue: number, count: number): number {
     return value > 1 ? 1 : value;
 }
 
+export function calculateQValue(sortedData: MutualExclusivity[]): MutualExclusivity[] {
```

Comment:
@jjgao can you review this logic for calculating q-value?

Reply:
agreed, it should be in its own file, and there should be more tests, including one with a longer input list.

I also think its crucial to link to a source in a comment.

Reply:
I realized that the q-value calcuation code was in Java for Enrichments (#5515). How about Co-exp, @adamabeshouse?

Reply:
coexp q-value is also on the server.

I still think its worthwhile to put it in its own file for future use

Reply:
@adamabeshouse agreed. We need to move the code to frontend soon for Enrichments. Maybe we could do it for Co-exp when we do that too.